server.js

```diff
@@ -7965,6 +7965,40 @@ cache(function (data, match, sendBadge, request) {
   });
 }));
 
+// Dependabot SemVer compatibility integration
+camp.route(/^\/dependabot\/semver\/([^/]+)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var package_manager = match[1];
+  var dependency_name = match[2];
+  var format = match[3];
+  var options = {
+    method: 'GET',
+    headers: { 'Accept': 'application/json' },
+    uri: `https://api.dependabot.com/badges/compatibility_score?package-manager=${package_manager}&dependency-name=${dependency_name}&version-scheme=semver`
+  };
+  var badgeData = getBadgeData('semver compatibility', data);
+  badgeData.links = [`https://dependabot.com/compatibility-score.html?package-manager=${package_manager}&dependency-name=${dependency_name}&version-scheme=semver`];
+  badgeData.logo = getLogo('dependabot', data);
+  request(options, function(err, res) {
+    if (err != null) {
```

Comment:
`checkErrorResponse` is a great way to handle errors. It's described [here](https://github.com/badges/shields/blob/master/doc/TUTORIAL.md).